core/src/main/scala/org/allenai/common/Logging.scala

```diff
@@ -46,5 +52,107 @@ trait Logging {
       if (internalLogger.isErrorEnabled) {
         internalLogger.error(message, throwable)
       }
+
+    /** Simple logback configuration.
+      * Hopefully this will be discoverable by just typing <code>logger.config().[TAB]</code>
+      *
+      * Examples:
+      * <code>
+      * logger.config("org.apache.spark").setLevel(Level.WARN)
+      *
+      * logger.config().addAppender(
+      *   logger.factory.patternLayoutEncoder("%-5level [%thread]: %message%n"),
+      *   logger.factory.consoleAppender
+      * )
+      * </code>
+      *
+      * @param loggerName the logger name, by default ROOT.
+      */
+    def config(loggerName: String = org.slf4j.Logger.ROOT_LOGGER_NAME) = {
+      new Config(loggerName)
+    }
+
+    class Config(loggerName: String = org.slf4j.Logger.ROOT_LOGGER_NAME) {
+      private val logger: Logger = LoggerFactory.getLogger(loggerName).asInstanceOf[Logger]
+
+      /** Resets the logger. */
+      def reset() = {
```

Comment:
Done. Added a fluent interface, so its a bit easier to do multiple things to one Config.